src/app/universe/LicenseSection.scss

```
@@ -0,0 +1,75 @@
+
+  &__link {
+    color: var(--universe-paper);
+    text-decoration: underline;
+    text-decoration-color: rgba(115, 209, 173, 0.58);
+    text-underline-offset: 0.2em;
+    transition: color 180ms ease, text-decoration-color 180ms ease;
+
+    &:hover,
+    &:focus-visible {
+      color: #73d1ad;
+      text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)